src/entities/factory/api/api-service.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiSettings, getApiDomainHeaders } from "@/shared/api";
+import { FactorySearchRq } from "@/entities/factory";
+import { PaginationState } from "@tanstack/react-table";
+
+type FactorySearchFilters = Partial<
```

Comment:
its more convinient to relocate to `./types` 
It will help to prevent the appearance of duplicates